### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by PedroDelCargo**
Hi,
I have added french translation.

### CODE REVIEW

**Comment by PiotrMachowski** [comment]
Hi! You have to add all changed files in one Pull Request (like here: #137)

**Comment by PedroDelCargo** [comment]
Hi,
I don't know how to do that...

Le mar. 16 nov. 2021 à 10:51, Piotr Machowski ***@***.***> a
écrit :

**Comment by PiotrMachowski** [comment]
You can create and merge following pull requests:

https://github.com/PedroDelCargo/lovelace-xiaomi-vacuum-map-card/compare/PedroDelCargo:patch-3...PedroDelCargo:patch-1?expand=1

https://github.com/PedroDelCargo/lovelace-xiaomi-vacuum-map-card/compare/PedroDelCargo:patch-3...PedroDelCargo:patch-2?expand=1

**Comment by PiotrMachowski** [comment]
That's better 👍

**Comment by PedroDelCargo** [comment]
I have created and merged pull requests. I hope this is what you expected.

Le mar. 16 nov. 2021 à 11:01, Piotr Machowski ***@***.***> a
écrit :

File: src/localize/languages/fr.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
+{
```
Comment (PiotrMachowski) [inline src/localize/languages/fr.js:1]: this file should be named `fr.json`

Comment (PiotrMachowski) [reply src/localize/languages/fr.js]: renamed

**Comment by PiotrMachowski** [comment]
Ok, everything is ok now 👍 , I will merge it when I will be preparing a new release

**Comment by PedroDelCargo** [comment]
Thank you. Sorry for my bad job! I'm not used to work with github...

Le mar. 16 nov. 2021 à 11:24, Piotr Machowski ***@***.***> a
écrit :

**Pull request merged**